platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -184,11 +188,9 @@ public static class ClassVersion {
      */
     private static final int MAX_FULL_REHASHING_TIMEOUT = 3600; // 1 hour
 
-    /**
-     * Placeholder (since this is such a hotspot) to hold the results from {@link ConfigurationHolder#getConfigData(Class)}
-     * rather than calling that method more than once during the lifecycle of a {@link VirtualRootNode} instance.
-     */
-    private final VirtualMapConfig config = ConfigurationHolder.getConfigData(VirtualMapConfig.class);
+    /** Platform configuration */
+    @NonNull
+    private final Configuration configuration;
```

Comment:
It looks like `VirtualRootNode` only needs a `VirtualMapConfig`, not the whole `Configuration` object. I don't know how expensive `Configuration.getConfigData()` calls are, but it probably makes sense to call it once in the constructor and change this field back to `VirtualMapConfig`

Reply:
Good catch!
Done: https://github.com/hashgraph/hedera-services/pull/15562/commits/8fa1dbc64ba51588dd4b8fa5ddc78ff4ae82e008